assets/files.wxs

```diff
@@ -3065,6 +3065,23 @@
       <Component Id="cmpEC42F4F98D70437BB0E290A0E0FE1775" Guid="{5483092a-12c2-4e6c-b730-be491b8909c0}">
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Reflection.Emit.dll; SideIndicator=<=}, @{InputObject=System.Threading.Timer.dll; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 127
127:         $diffs | Should -BeExactly $null
```